main.py

```diff
@@ -19,7 +19,7 @@
 
 
 def default_experiment():
-    return {'epochs': 2,
+    return {'epochs': 100000,
```

Comment:
100k is huuuuuuge is that really what they use ? 2 was for testing, but 100 should be fine, no ?

Reply:
Yeah they use 100 000. Get ready to burn some GPUs :) - I agree that 100 is fine for testing, we can just set it to 100 with the CLI.

Reply:
daaaamn ! Can' compete with deepmind haha. Tht explains the low learning rate then. Maybe let's use 100 in cli and 1000 or 100k in the experiments

Reply:
I made this change in the latest commit.